tests/test_reading_api.py

```diff
@@ -28,6 +28,16 @@ def test_get_success(mocked_client):
     assert "<judgment>" in response.text
 
 
+@patch("openapi_server.apis.reading_api.client_for_basic_auth")
+def test_passes_unpublished_if_true(mocked_client):
```

Comment:
[suggestion] docstring to clarify the setup

Reply:
Added